packages/editor/src/components/parts/program/editor/Editor.css

```diff
@@ -0,0 +1,6 @@
+.message {

```

Comment:
please invent a new unique class name.
Css is normally not scoped, which means the CSS of the message widget now is active here (and vice versa), and if we change this one or the other it can also change the styles of the message widget...